src/components/Sites/routes/_SharedComponent/ProjectTopCard/ProjectTopCard.tsx

```diff
@@ -132,187 +132,192 @@ const ProjectTopCard = () => {
-                  target="_blank"
-                  rel="noopener noreferrer"
-                >
-                  <img
-                    className="deployment-screenshot"
-                    src={imageUrl(
-                      selectedProject?.latestDeployment?.screenshot?.url,
-                    )}
-                    // onClick={latestDeployment?.sitePreview}
-                    alt={"Preview not Available"}
-                  />
-                </a>
-              ) : (
-                <Skeleton height={200} width={320} duration={2} />
-              )}
-            </div>
-            <div className="deployment-left">
-              <div>
-                <div className="project-top-card-header">
-                  <h2 className="project-top-card-header-title">
-                    {!projectLoading ? (
-                      selectedProject?.name
-                    ) : (
-                      <Skeleton width={200} duration={2} />
-                    )}
-                  </h2>
-                  <p className="project-top-card-header-description">
-                    {!projectLoading ? (
-                      <>
-                        <u>Production</u>:{" "}
-                        {selectedProject?.latestDeployment?.configuration.branch} -
-                        Last published at {lastPublishedDate}
-                      </>
+        <div className="deployment-top-card-container">
+          <div className="deployment-screenshot-container">
+            {!projectLoading ? (
+              <a
+                href={latestDeployment?.sitePreview}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <img
+                  className="deployment-screenshot"
+                  src={imageUrl(selectedProject?.latestDeployment?.screenshot?.url)}
+                  // onClick={latestDeployment?.sitePreview}
+                  alt={"Preview not Available"}
+                />
+              </a>
+            ) : (
+              <Skeleton height={200} width={320} duration={2} />
+            )}
+          </div>
+          <div className="deployment-right">
+            <div className="project-top-card-header">
+              <div className="project-top-card-header-title-container">
+                <h2 className="project-top-card-header-title">
+                  {!projectLoading ? (
+                    selectedProject?.name
+                  ) : (
+                    <Skeleton width={200} duration={2} />
+                  )}
+                </h2>
+                <div className="archive-tag-container">
+                  {!projectLoading ? (
+                    selectedProject?.state === "ARCHIVED" ? (
+                      <span className="archive-tag">{selectedProject?.state}</span>
                     ) : (
-                      <Skeleton width={400} duration={2} />
-                    )}
-                  </p>
+                      <></>
```

Comment:
```suggestion
                      null
```
Put this if you don't have anything to show